modules/manage/pages/audit-logging.adoc

```diff
@@ -23,14 +23,15 @@ Messages recorded to the audit log topic comply with the https://schema.ocsf.io/
+* xref:reference:cluster-properties.adoc#audit_client_max_buffer_size[`audit_client_max_buffer_size`]: Integer value defining the number of bytes allocated by the internal audit client for audit messages. When changing this, you must disable audit logging and then re-enable it for the change to take effect. Consider increasing this if your system generates a very large number of audit records in a short amount of time. Default: `16777216`.
+* xref:reference:cluster-properties.adoc#audit_queue_max_buffer_size_per_shard[`audit_queue_max_buffer_size_per_shard`]: Integer value defining the maximum amount of memory in bytes used by the audit buffer in each shard. Once this size is reached, requests to log additional audit messages will return a non-retryable error. You must restart the cluster when changing this value. Default: `1048576`.
+* xref:reference:cluster-properties.adoc#audit_enabled_event_types[`audit_enabled_event_types`]: List of strings in JSON style identifying the event types to include in the audit log. This may include any of the following - `management, produce, consume, describe, heartbeat, authenticate, schema_registry, admin`. Default: `'["management","authenticate","admin"]'`.
+* xref:reference:cluster-properties.adoc#audit_exclude_topics[`audit_exclude_topics`]: List of strings in JSON style identifying the topics the audit logging system should ignore. This list cannot include the `__audit_log` topic. Redpanda will reject the command if you do attempt to include that topic. Default: null.
+* xref:reference:cluster-properties.adoc#audit_queue_drain_interval[`audit_queue_drain_interval`]: Internally, Redpanda batches audit log messages in memory and periodically writes them to the audit log topic. This defines the period in milliseconds between draining this queue to the audit log topic. Longer intervals may help prevent duplicate messages, especially in high throughput scenarios, but they also increase the risk of data loss during hard shutdowns where the queue is lost. Default: `500`.
+* xref:reference:cluster-properties.adoc#audit_exclude_principals[`audit_exclude_principals`]: List of strings in JSON style identifying the principals the audit logging system should ignore. Principals can be listed as `User:name` or `name`, both are accepted. Default: null.
 
 Even though audited event messages are stored to a specialized immutable topic, standard topic settings still apply. For example, you can apply the same Tiered Storage, retention time, and replication settings available to normal topics. These particular options are important for controlling the amount of disk space utilized by your audit topics.
```

Comment:
At the bottom of this page is a leftover placeholder `SPECIFY AN EXCESSIVELY LONG VALUE HERE`.

Reply:
I have no clue how that got through review. Thanks for catching that!